packages/amplication-server/src/core/project/project.service.ts

```diff
@@ -94,7 +94,7 @@ export class ProjectService {
     await this.billingService.reportUsage(
       project.workspace.id,
       BillingFeature.Services,
-      archivedServiceCount
+      -archivedServiceCount
```

Comment:
what is this `-` ?

Reply:
I understand, but it looks odd 🤕 
If I'm not mistaken, every time a user deletes a project we send the **total** `archivedServiceCount`. That means that we will subtract more than one time the same services.
Am I missing something?